android/build.gradle

```
@@ -50,8 +51,8 @@ android {
         disable 'GradleCompatible'
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
     }
 }
```

I think we can keep this at 1.8 if we specify a `jvmToolchain` (which if I understand correctly, dictates the version of Java that Gradle itself uses), and set the `jvmTarget` to match the `targetCompatibility`:

```suggestion
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }
    kotlinOptions {
        jvmTarget = "1.8"
    }
}

kotlin {
    jvmToolchain(17)
}
```